pkg/hookbot/hookbot.go

```diff
@@ -308,3 +308,19 @@ func (h *Hookbot) ServeSubscribe(conn *websocket.Conn, r *http.Request) {
 		}
 	}
 }
+
+func IsConnectionClose(err error) bool {
```

Comment:
I wasn't bleching at your code; I was bleching at Go for making this the least horrible solution that reliably works.